docs/source/index.rst

```diff
@@ -5,40 +5,40 @@ StreamAlert
   :align: center
   :alt: StreamAlert
 
-StreamAlert is a serverless, realtime data analysis framework which empowers you to ingest, analyze, and alert on data from any environment, using datasources and alerting logic you define.
+StreamAlert is a serverless, real-time data analysis framework which empowers you to ingest, analyze, and alert on data from any environment, using data sources and alerting logic you define.
+Security teams use StreamAlert to scan terabytes of log data every day for proactive threat detection and prevention.
```

Comment:
I'd caution against saying `and prevention` here. SA doesn't do any prevention, and acts solely as an intrusion detection system (IDS) vs intrusion prevention system (IPS). you could, however, say `detection and response`